<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1125</width>
    <height>645</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1125</width>
    <height>645</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1125</width>
    <height>645</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>RecipeBook</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../src/default.jpg</normaloff>../src/default.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow {background: rgb(252, 238, 251);}
QPushButton {
	background: rgb(239, 187, 238);
	border-radius: 10px;
	border-width: 0px;}
QPushButton:hover {
	background: rgb(244, 204, 242);
	border-width: 0px;
	border-radius: 10px;}
QLineEdit {background: rgb(239, 187, 238);}
#nameEdit {border-radius: 15px;
	border-width: 1px;
	min-height: 30px;
	max-height: 30px;
	min-width: 250px;
	max-width: 250px;}
#searchBox {
	border-radius: 5px;
	border-width: 1px;
	min-height: 30px;
	max-height: 30px;
	min-width: 250px;
	max-width: 250px;
	 }
#recipesTable {background: rgb(252, 238, 251);}
#outputBox  {}


</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="bestButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri Light</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>☆</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>250</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>250</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Calibri Light</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="outputBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>90</y>
      <width>631</width>
      <height>531</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri Light</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="categoriesButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>161</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri Light</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Категория</string>
    </property>
   </widget>
   <widget class="QTableView" name="recipesTable">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>200</width>
      <height>521</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri Light</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>10</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri Light</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>➕ Добавить новый рецепт</string>
    </property>
   </widget>
   <widget class="QPushButton" name="edit_bestButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>480</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri Light</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>☆</string>
    </property>
   </widget>
   <widget class="QLabel" name="picLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>90</y>
      <width>220</width>
      <height>220</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="editButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>380</y>
      <width>220</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri Light</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Изменить рецепт</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>430</y>
      <width>220</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri Light</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Удалить рецепт</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>330</y>
      <width>220</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>220</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>220</width>
      <height>40</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Calibri Light</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="filterButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri Light</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Фильтры</string>
    </property>
   </widget>
   <widget class="QPushButton" name="helpButton">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri Light</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>?</string>
    </property>
   </widget>
   <zorder>bestButton</zorder>
   <zorder>outputBox</zorder>
   <zorder>categoriesButton</zorder>
   <zorder>recipesTable</zorder>
   <zorder>addButton</zorder>
   <zorder>edit_bestButton</zorder>
   <zorder>picLabel</zorder>
   <zorder>editButton</zorder>
   <zorder>deleteButton</zorder>
   <zorder>nameLabel</zorder>
   <zorder>searchBox</zorder>
   <zorder>filterButton</zorder>
   <zorder>helpButton</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
